app/logic/serviceLearningCourses.py

```diff
@@ -77,17 +77,47 @@ def saveCourseParticipantsToDatabase(cpPreview: Dict[str, Dict[str, Dict[str, Li
                 print(f"Unable to save course {course}. {courseInfo['errorMsg']}")
                 continue
 
-            courseObj: Course = Course.get_or_create(
-                 courseAbbreviation = course,
-                 term = termObj, 
-                 defaults = {"CourseName" : "",
-                             "sectionDesignation" : "",
-                             "courseCredit" : "1",
-                             "term" : termObj,
-                             "status" : 4,
-                             "createdBy" : g.current_user,
-                             "serviceLearningDesignatedSections" : "",
-                             "previouslyApprovedDescription" : "" })[0]
+            termCheck = Term.get(Term.id == termObj)
+
+            checkTermOrder = termCheck.termOrder
+
+            check_for_Existing_Courses = list((Course.select()
```

Comment:
This variable doesn't fit our coding style (or Python's)

Reply:
Also, variables names should be about the data they hold, rather than the action it took to get the data. Noun rather than verb

Reply:
We renamed it to "existingCourses"